### ISSUE TITLE
Figure out how ActionEntity linking permissions work

### ISSUE THREAD

**Issue by zelerin** [mutgos/mutgos_server#20]
Right now there is no design for how permissions for linking an ActionEntity to an Entity work.  The design needs to be hashed out and implemented.

**Comment by zelerin** [comment]
I am thinking this could be accomplished by adding a 'link' flag to the security flag enum, similar to how MUCKs do it.

Related, I'm not sure if a chown flag (allowing someone else to take ownership of something) is a good idea, or if just setting them as an admin would work.  There could still be a chown command, it will work if the target is an admin of the Entity rather than a specific security flag being set.

In short, I don't want to have a myriad of flags if I can help it; I'm trying to keep security more consistent and with less to remember about how it works.
